EventListener/LeadSubscriber.php

```diff
@@ -65,7 +65,7 @@ public function doAutoRunEnhancements(LeadEvent $event)
                     '|',
                     [
                         $lead->getFirstname(),
-                        $lead->getLastActive(),
+                        $lead->getLastActive()->format('Y-m-d H:i:s'),
```

Comment:
Thanks for your help Joey
This fails in tests because $lead->getLastActive() is null in some cases.
Could you try this?
```
($lead->getLastActive() ? $lead->getLastActive()->format('Y-m-d H:i:s') : ''),
```

Reply:
Updated!